app/jobs/export_collection_pdf_job.rb

```diff
@@ -11,9 +10,23 @@ def perform(resource_id, logger: Logger.new(STDOUT))
         next
       end
 
-      filename = "#{member.source_metadata_identifier.first.gsub(/.*_/, '')}.pdf"
-      logger.info "Exporting #{member.id} as #{filename}"
-      ExportService.export_pdf(member, filename: filename)
+      fn = member.source_metadata_identifier.first.gsub(/.*_/, "")
+      member.decorate.volumes.empty? ? export_file(member, fn) : export_volumes(member, fn)
```

Comment:
I'm thinking we should push this to ExportService so that the other id-based rake task could use it, also.

Reply:
👍